<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindow</class>
 <widget class="QMainWindow" name="DataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>781</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>901</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>247</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Таблица базы данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Resetfilters">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>610</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить все фильтры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sorting">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Filter">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>580</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтр</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="Table_Gr_prog"/>
    <addaction name="Table_gr_konk"/>
    <addaction name="Table_VUZ"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Анализ</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Финансирование</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Выход</string>
    </property>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="Table_Gr_prog">
   <property name="text">
    <string>Информация по проектам, выполняемым по конкурсам грантов</string>
   </property>
  </action>
  <action name="Table_gr_konk">
   <property name="text">
    <string>Информация по конкурсам</string>
   </property>
  </action>
  <action name="Table_VUZ">
   <property name="text">
    <string>Информация по ВУЗам</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Справка по программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
